src/i18n.js

```diff
@@ -21,6 +21,7 @@ i18n
     resources: {
       en: {
         translations: {
+            "deviceVerificationSystem": "Device Verification System",
```

Comment:
It should be "Device Verification Subsystem"

Reply:
Corrected